athena/plotting/visualization.py

```diff
@@ -57,6 +50,8 @@ def spatial(so, spl: str, attr: str, *, mode: str = 'scatter', node_size: float
         show: whether to show the plot or not
         save: path to the file in which the plot is saved
         tight_layout: whether to apply tight_layout or not.
+        filter_col: string identifying the column in so.obs to use to filter ot cells that should not be plotted.
+        labels: list of strings that identify the cells that should be included in the plot. These should be entries in filter_col.
```

Comment:
filter_col and labels are both input to the filter logic. Maybe one could rename this in a way that is is clear for the user that they belong together just from the argument name, like filter_col filter_label

Reply:
I think its a good idea. However this will take some time to change since "labels" is called in a number of places. But I will do it :)

Reply:
Lets discuss this in the group meeting if you join. What IDE do you use? With pycharm you can start a refactor session that speeds up such refactoring dramaticallly

Reply:
I use VSCode. Maybe it has some built-in functionality that can be used.

Reply:
`include_labels`